### PR TITLE
bio update Akshay-bendadi

### PR DESCRIPTION
**Pull request by Akshay-bendadi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2181"></a>

### CODE REVIEW

File: public/data/Akshay-bendadi.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Akshay Bendadi",
   "type": "personal",
-  "bio": " computer science student | DevOps and Open-Source Enthusiast ", 
+  "bio": " DevOps and Open-Source Enthusiast| Learning in Public   ",
```
Comment (schmelto) [inline public/data/Akshay-bendadi.json:4]:
```suggestion
  "bio": " DevOps and Open-Source Enthusiast| Learning in Public", 
```

File: public/data/Akshay-bendadi.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Akshay Bendadi",
   "type": "personal",
-  "bio": " computer science student | DevOps and Open-Source Enthusiast ", 
+  "bio": " DevOps and Open-Source Enthusiast| Learning in Public",
```
Comment (schmelto) [inline public/data/Akshay-bendadi.json:4]:
```suggestion
  "bio": "DevOps and Open-Source Enthusiast| Learning in Public", 
```

**Review by schmelto — APPROVED** [review]

**Comment by Akshay-bendadi** [comment]
i'm still not able to see the change

**Comment by eddiejaoude** [comment]
@Akshay-bendadi it is merged and deploy, are you sure you can't see the changes?

**Comment by Akshay-bendadi** [comment]
image
image

**Comment by eddiejaoude** [comment]
I am not sure why, please submit a new PR with the changes

**Pull request merged**